Adding security considerations (#38)
pyansys/dev-guide: doc/source/coding_style/best_practices.rst

The second approach is more readable and better documented. Additionally,
you could implement a unit test for ``is_consonant``.

Security Considerations
~~~~~~~~~~~~~~~~~~~~~~~

> Security is is an ongoing process involving people and practices, and ensures application confidentiality, integrity, and availability [#]_.

Reviewer: ```suggestion
Security, an ongoing process involving people and practices, ensures application confidentiality, integrity, and availability [#]_.
```

Any library should be secure and implement good practices that avoid or mitigate possible security risks.

> This is especially relevant in libraries that requests user input (such web-services).

Reviewer: ```suggestion
This is especially relevant in libraries that request user input (such as web services).
```

> Security is a very wide topic hence we do recommend to have a look at the following useful resources:

Reviewer: ```suggestion
Because security is a very wide topic, we recommend you review these useful resources:
```
